src/screens/TopicPageScreen/MemoizedListComponent.js

```diff
@@ -51,6 +54,7 @@ const RenderListFeed = (props) => {
     navigateToLinkContextPage,
     getHeightFooter,
     getHeightHeader,
+    getTotalReaction,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `getTotalReaction` has been added to the list of destructured props. Make sure this function is provided as a prop when using this component.
```diff
+    getTotalReaction,
     statusDownvote,
     voteStatus,
     checkVotes,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->